tests/slop/slop1.test.ts

```diff
@@ -47,5 +47,22 @@ describe("Slop Tests", () => {
     } else {
       expect(failures.length).toBe(0) // Assert that there are no failures
     }
-  })
+  }),
+    it("should throw an error for invalid footprint function", async () => {
+      const { fp } = await getTestFixture("slop1")
+
+      const invalidFootprint = "invalidFunction"
+
+      try {
+        fp.string(invalidFootprint).soup()
+        expect.fail("Expected an exrror to be thrown, but no error was thrown")
+      } catch (error) {
+        expect(error).toBeInstanceOf(Error)
+
+        expect(error.message).toContain("Function not found for footprinter")
+        expect(error.message).toContain(
+          "Specify a valid function like .dip, .lr, .p, etc.",
+        )
+      }
+    })
```

Comment:
why did you add this to a slop test? can you move it to a different test file? Also you should request me for review to  get a review faster